tests/e2e/cypress/cypress/integration/circulation/checkout-checkin.spec.js

```diff
@@ -66,9 +66,9 @@ describe('Circulation checkout checkin', function() {
       .should('have.class', 'fa-arrow-circle-o-down')
 
     // Go back to homepage, search document and delete item
-    cy.goToItem(itemBarcode)
+    cy.goToDocumentDetailView(itemBarcode)
     // click on Delete button
-    cy.get('#item-' + itemBarcode + '-delete', {timeout: timeOut}).click()
+    cy.get('#item-' + itemBarcode + ' > .col-sm-4 > .btn-outline-danger', {timeout: timeOut}).click()
```

Comment:
Not for now remarks ::

I'm not fan about using "classes selector" and ">".

First, I'm pretty sure, you can simplify your selection using `'#item-' + itemBarcode + '.btn-outline-danger'` :: it will be better if in the future we change the column width.

After, for me, the best solution should be to add a `role` attribute to button (in the UI ; yes a lot of change) and use this attribute to select the correct button. The selection could looks like `'#item-' + itemBarcode + 'button[@role="delete"]`

Reply:
I didn't adapt ui for the moment, but sure, this should be improved.